data/json/recipes/weapon/ranged.json

```diff
@@ -1230,7 +1230,7 @@
     "components": [
       [ [ "2x4", 8 ], [ "stick", 16 ] ],
       [ [ "rope_superior_short", 2, "LIST" ] ],
-      [ [ "foot_crank", 1 ] ],
+      [ [ "simple_crank", 1 ], [ "foot_crank", 1 ] ],
```

Comment:
I'm not convinced the simple crank could handle the pull weight. I'm also not convinced it wouldn't severely slow down usage.

Reply:
Hmm never mind didn't realize it was forged.